### ISSUE TITLE
I can't sign out, change password, update terminal so I can download packages

### ISSUE THREAD

**Issue by shebeary** [microsoft/vscode#99566]
Issue Type: <b>Bug</b>

I can't sign out, change password, update terminal so I can download packages

npm install express
7 vulnerabilitys
npm install -g npm
rejected
too many changes in git
manage extention
nothing in link
update terminal
chsh -s /bin/zsh
password
which one?
don't know!
any way to check or know which one?
no way to sign out of vsc to try and figure it out
help!

VS Code version: Code 1.45.1 (5763d909d5f12fe19f215cbfdd29a91c0fa9208a, 2020-05-14T08:33:47.663Z)
OS version: Darwin x64 19.0.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4258U CPU @ 2.40GHz (4 x 2400)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|12, 15, 16|
|Memory (System)|4.00GB (0.08GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
LiveServer|rit|5.6.1
bootstrap4-vscode|the|6.1.0

</details>

**Comment by joaomoreno** [comment]
I'm sorry, but it's *really* hard to understand what the issue is. Can you please explain it in plain English? Thanks!
